sdk/identity/azure-identity/src/main/java/com/azure/identity/InteractiveBrowserCredential.java

```diff
@@ -119,8 +119,9 @@
     private AccessToken updateCache(MsalToken msalToken) {
         cachedToken.set(
                 new MsalAuthenticationAccount(
-                        new AuthenticationRecord(msalToken.getAuthenticationResult(),
-                                identityClient.getTenantId(), identityClient.getClientId())));
+                    new AuthenticationRecord(msalToken.getAuthenticationResult(),
+                                identityClient.getTenantId(), identityClient.getClientId()),
+                    msalToken.getAccount().getTenantProfiles()));
```

Comment:
nit: The formatting looks a bit odd. Probably moving the instantiation of `AuthenticationRecord` outside of `set()` would make it more readable too.

Reply:
agreed, 
will update it.